docs/ref/contrib/admin/filters.txt

```diff
@@ -0,0 +1,162 @@
+=================
+Admin List Filter
+=================
+
+.. currentmodule:: django.contrib.admin
+
+.. attribute:: ModelAdmin.list_filter :noindex:
```

Comment:
I think this `attribute` declaration (still) belongs back in the reference. 

This doc is providing details on the allowed values for ``` :ref:`ModelAdmin.list_filter` ```
It's a free standing doc, so it probably wants a introductory paragraph saying such. 

Then we can outdent the list items below. The only reason they're presented as a list, I think, is that they had to fit in the context of the larger doc. Now they're independent, it can be restructured. 

How about an overview stating the options (as a list say) and then sections for each possibility, which allows us to expand with the examples we'll need for this PR. 

I think moving to it's own page, plus small restructuring like this, could be merged as a single commit before adjusting the content for the new functionality.

Reply:
- Removed the `attribute` declaration from filters.txt
- Added an overview stating the options for allowed values for `ModelAdmin.list_filter` in a numbered list.
- I have removed the details of the examples related to this PR and will add them after these changes are finalized.